generators/app/templates/client/app/components/date-interceptor/date-interceptor.js

```diff
@@ -1,9 +1,10 @@
 (function () {
   'use strict';
+  var regexIso8601 = /^(\d{4}|\+\d{6})(?:-(\d{2})(?:-(\d{2})(?:T(\d{2}):(\d{2}):(\d{2})\.(\d{1,})(Z|([\-+])(\d{2}):(\d{2}))?)?)?)?$/;
```

Comment:
This would apply for simple strings like `1234`, which would be converted to a Date object.
What is the \+ part about, could not find in in the ISO 8601 spec?
![regexp](https://cloud.githubusercontent.com/assets/1121875/15603094/2729783c-23f9-11e6-92c2-1141d8a887c2.png)